nodeadm/internal/kubelet/config.go

```diff
@@ -119,6 +122,7 @@ func defaultKubeletSubConfig() kubeletConfig {
 		CgroupRoot:               "/",
 		ClusterDomain:            "cluster.local",
 		ContainerRuntimeEndpoint: containerd.ContainerRuntimeEndpoint,
+		EnforceNodeAllocatable:   []string{"pods"},
```

Comment:
This is the default -- just making it explicit here that we aren't enforcing either kube or system reserved.